src/test/util_string_tests.cpp

```diff
@@ -11,13 +11,25 @@ using namespace util;
 
 BOOST_AUTO_TEST_SUITE(util_string_tests)
 
+template <unsigned NumArgs>
+std::string TfmFormatZeroes(const char* fmt)
+{
+    return std::apply([fmt](auto... args) {
+        return tfm::format(std::string{fmt}, args...);
+    }, std::tuple_cat(std::array<int, NumArgs>{}));
+}
+
 // Helper to allow compile-time sanity checks while providing the number of
 // args directly. Normally PassFmt<sizeof...(Args)> would be used.
 template <unsigned NumArgs>
 inline void PassFmt(ConstevalFormatString<NumArgs> fmt)
 {
-    // This was already executed at compile-time, but is executed again at run-time to avoid -Wunused.
+    // Exercise happy paths at run-time for code coverage metrics.
     decltype(fmt)::Detail_CheckNumFormatSpecifiers(fmt.fmt);
+
+    // Prove parity with tinyformat, which will throw if the number of
+    // conversion specifiers is incorrect.
```

Comment:
In commit "test: Prove+document ConstevalFormatString/tinyformat parity" (db081bd66015f7899696e6eb4a76052446685b64)

Can there be a test here or somewhere in this file that verifies calls to format really do throw if the number of specifiers is wrong? I am concerned there could be a future change like #30928 where tinyformat errors start being handled differently and these tests become broken without someone noticing.